doc/nrf/known_issues.rst

```diff
@@ -1357,90 +1387,104 @@ OCT-2501: Charging over seven hours results in error
 
   **Workaround:** To start the charging again, turn the nRF5340 Audio DK off and then on again.
 
-.. rst-class:: v2-3-0
+.. rst-class:: v2-4-0
 
-OCT-2472: Headset fault upon gateway reset in the bidirectional stream mode
-  The headset may react with a usage fault when the :kconfig:option:`CONFIG_STREAM_BIDIRECTIONAL` application Kconfig option is set to ``y`` and the gateway is reset during a stream.
-  This issue is under investigation.
+OCT-2539: Presentation delay may not work as expected under some configurations.
+  Relates to OCT-2557, OCT-2489.
+  OCT-2584: Initial L-R sync may lock with an offset.
```

Comment:
Wrong issue number - 2584 is "Controller path is wrong one place in the documentation"
Should this be 2358?
Should this be grouped with the OCT-2401 above?

Reply:
Good catch, corrected.
No, they are unrelated. One is HW the other is FW.